components/tally/tally.app.mjs

```diff
@@ -1,11 +1,74 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "tally",
-  propDefinitions: {},
+  propDefinitions: {
+    forms: {
+      label: "Form",
+      description: "Select a form",
+      type: "string",
+      async options() {
+        const forms = await this.getForms();
+
+        return forms.map((form) => ({
+          label: form.name,
+          value: form.id,
+        }));
```

Comment:
Is pagination available for this request?

Reply:
I didn't see pagination for the forms in the documentation(or internal platform requests).